doc/README.yaml.md

```diff
@@ -0,0 +1,240 @@
+
+# README for Open-CE build YAML files
```

Comment:
These should be titled "Open-CE Environment files". YAML can be left out of the title.

Reply:
Done (here and universally).